master_db.py

```diff
@@ -54,16 +57,28 @@
                                         "name": "nsample", "type":["int", "List[int]"]}}]  
                 }
 
-    for name in distributions:
+    print("Insertando distribuciones...")
+    for name, parameters in distributions.items():
         try:
-            distribution = Distribution()
-            distribution.name = name
-            distribution.type = parameters[name]
+            distribution = MasterDistribution(
+                name=name,
+                type=parameters).save()
 
-            distribution.save()
+            print(f"distribución {name} insertada correctamente")
+        except Exception as error:
+            print(f"No fue posible insertar la distribución {name}")
+            print(error)
+    
 
-            print(f"{name} insertada correctamente")
+    print("/n Insertando estados de enfermedad...")
+    for name in disease_status:
+        try:
+            MasterDiseaseStates(
+                identifer=uuid1().hex, 
+                name=name
+            ).save()
+            
+            print(f"Estado {name} insertado correctamente")
         except Exception as error:
-            print(f"No fue posible insertar {name}")
+            print(f"No fue posible insertar el estado {name}")
             print(error)
```

Comment:
Same, delete all prints,

If you need set some flag, could be better set a comment